fhir-install/docker/copy-dependencies-db2-migration.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -ex
+
+SCHEMA_VERSION="4.0.1"
```

Comment:
We should document this needs to be updated in DB2SchemaMigration.md

Reply:
I added it to a new "Testing migrations" section in that document